addons/io_scene_gltf2/__init__.py

```diff
@@ -353,7 +353,7 @@ def invoke(self, context, event):
             preferences = bpy.context.preferences
         for addon_name in preferences.addons.keys():
             try:
-                if hasattr(sys.modules[addon_name], 'glTF2ExportUserExtension'):
```

Comment:
I think this guard should rather check if the module hasattrs at least one of `glTF2ExportUserExtension` or `glTF2ExportUserExtensions`.

The name `register_panel` alone is too generic to assume that any addon that just has that is intending for us to call it.